cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -58,7 +58,7 @@ type describeClusterOptions struct {
 	showOtherConditions string
 	showMachineSets     bool
 	disableNoEcho       bool
-	disableGrouping     bool
```

Comment:
You can't remove this variable until the flag is actually removed

Reply:
added it back